### ISSUE TITLE
Wildfly 23 not working in extension

### ISSUE THREAD

**Issue by robstryker** [redhat-developer/vscode-server-connector#487]
Wildfly 23 is misidentified as EAP 6.0 in current extension. 

Wildfly 23 support was added to rsp-server:
* d9287f57 - Fixes #633 - add wf23 support (5 weeks ago) <Rob Stryker>

EAP 7.4 was added to rsp-server: 
* b0bdb2b4 - (HEAD -> master, origin/master, origin/HEAD) EAP 7.4 (3 weeks ago) <Rob Stryker>

0.23.11 of rsp-server was "released":  04-Mar-2021 17:12

vscode-server-connector was last released:   2/15/2021, 11:08:40 AM

In short, wfly 23 and eap 7.4 are not available in current extension. This must be fixed ASAP.

**Comment by odockal** [comment]
@robstryker What is state of this issue? I have not noticed anything wrong when creating wf 23 from the disc.

**Comment by odockal** [comment]
Or eap in that matter: "org.jboss.tools.rsp.server.typeId": "org.jboss.ide.eclipse.as.eap.74",

**Comment by robstryker** [comment]
new version was released so this is probably fixed.
